contracts/token/ERC721/ERC721Mintable.sol

```diff
@@ -0,0 +1,71 @@
+pragma solidity ^0.4.24;
+
+import "./ERC721.sol";
+import "../../access/roles/MinterRole.sol";
+
+
+/**
+ * @title ERC721Mintable
+ * @dev ERC721 minting logic
+ */
+contract ERC721Mintable is ERC721, MinterRole {
+  event Minted(address indexed to, uint256 tokenId);
+  event MintingFinished();
+
+  bool public mintingFinished = false;
```

Comment:
public? Nooo

Reply:
good point; if we update this, also update the `ERC20Mintable` that this was copy/pasted from